src/reactor.cc

```diff
@@ -112,8 +112,11 @@ void Reactor::Tick() {
     // record the last time series entry if the reactor was operating at the
     // time of retirement.
     if (exit_time() == context()->time()) {
-      if (cycle_step > 0 && cycle_step <= cycle_time &&
-          core.count() == n_assem_core) {
+      if (cycle_step > 0 && cycle_step < cycle_time && 
+        core.count() == n_assem_core) {
+        cyclus::toolkit::RecordTimeSeries<cyclus::toolkit::POWER>(this, power_cap);
+      }  else if (refuel_time == 0 && cycle_step <= cycle_time && cycle_step > 0 &&
+                core.count() == n_assem_core)  {
```

Comment:
GCSG: same indentation issue.

Reply:
Also, couldn't this be simplified to remove the `cycle_step <= cycle_time && cycle_step > 0` part, ie just `refuel_time == 0  && core.count() == n_assem_core`.  That is, if we have fuel and can refuel instantly, we will produce power.

Reply:
Thanks! that is done!